recipes/mp-units/2.3.0/test_package/conanfile.py

```diff
@@ -13,20 +13,24 @@ class TestPackageConan(ConanFile):
     def requirements(self):
         self.requires(self.tested_reference_str)
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=4.2.1 <5]")
+
     def layout(self):
         cmake_layout(self)
 
     def generate(self):
         tc = CMakeToolchain(self)
         opt = self.dependencies["mp-units"].options
+        tc.cache_variables["MP_UNITS_BUILD_CXX_MODULES"] = opt.cxx_modules
```

Comment:
Hi @mpusz,

This cmake variable does not have any effect in the test_package, so I don't understand the purpose of its usage here. Also, it is generating a warning in the build, which also points to other variables used in the same way:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    MP_UNITS_API_CONTRACTS
    MP_UNITS_API_STD_FORMAT
    MP_UNITS_BUILD_CXX_MODULES
```

My suggestion is to remove this line and take a look at the rest of the variables that are not needed, in order to keep the test package recipe as simple as possible for future maintenance (it can be done in a future PR if desired).

```suggestion
```